### ISSUE TITLE
Always decoding, even with random string

### ISSUE THREAD

**Issue by espindl** [sqids/sqids-dotnet#35]
var sqids = new SqidsEncoder<long>(new SqidsOptions { Alphabet = _sqidAlphabet });
return sqids.Decode(sqid);

This code always decode, even with random string. But the [playground](https://sqids.org/playground) returns Error: Invalid ID with the same string. Am I doing something wrong or missing a very primitive concept?

**Comment by 4kimov** [comment]
Behind the scenes, the playground is simply doing `id == encode(decode(id))` and if it's not equal, it'll output a user-friendly error.

More info: https://sqids.org/faq#valid-ids

**Comment by aradalvand** [comment]
@espindl  Also note that this has been explained in the README [here](https://github.com/sqids/sqids-dotnet#ensuring-an-id-is-canonical).

**Comment by espindl** [comment]
Yeah, thank you and sorry I missed this in both faq and readme @aradalvand @4kimov
